.pre-commit-config.yaml

```diff
@@ -31,13 +31,3 @@ repos:
         args: [--config=src/bentoml/grpc/buf.yaml, src/bentoml/grpc]
       - id: buf-lint
         args: [--config=src/bentoml/grpc/buf.yaml, --error-format=msvs, src/bentoml/grpc]
-  - repo: local
```

Comment:
The script itself won't raise error

Reply:
At least it complains 855 errors on my side...

Reply:
Eh I think at some point we will need to fix these, I think just disabling it now is ok. Can you just comment these instead of removing it?